numbat/modules/core/strings.nbt

```diff
@@ -3,27 +3,35 @@ use core::functions
 use core::error
 
 @description("The length of a string")
+@example("str_length(\"Numbat\")")
 fn str_length(s: String) -> Scalar
 
 @description("Subslice of a string")
+@example("str_slice(\"Numbat\", 0, 2)")
```

Comment:
Maybe use different arguments here to show that the second argument is not the length, but the end-index:
```suggestion
@example("str_slice(\"Numbat\", 3, 6)")
```